src/components/DiffView/index.spec.tsx

```diff
@@ -82,12 +93,14 @@ describe(__filename, () => {
     return { _fetchLinterMessages, dispatch, root, fakeThunk };
   };
 
-  const setUpLinterMessagesForPath = ({
+  const dispatchLinterMessages = ({
```

Comment:
In the past, we used the name of the action prefixed with an underscore (`_loadLinterResult`) for such functions.

Reply:
Since this is a direct wrapper around `loadLinterResult` then, yeah, I guess that would make sense.